CONTRIBUTING.md

```diff
@@ -129,88 +123,61 @@ You can get these values by registering a new integration on [Cisco Webex for De
 | `WEBEX_CONVERSATION_CLUSTER_SERVICE` | Service identifier used to lookup conversation servers in hostmap                                                        | identityLookup                                      |
 | `WEBEX_CONVERSATION_DEFAULT_CLUSTER` | Cluster used to convert from "us" cluster to actual cluster                                                              | urn:TEAM:us-east-2_a:identityLookup                 |
 | `WEBEX_LOG_LEVEL`                    | Maximum log level that should be printed to the console.                                                                 | log                                                 |
 | `WEBEX_REDIRECT_URI`                 | The URI to redirect to after authorization                                                                               | undefined                                           |
 | `WEBEX_SCOPE`                        | The Webex scope the users will authorize with                                                                            | undefined                                           |
-| `WDM_SERVICE_URL`                    | The WDM service url before the catalog is downloaded                                                                     | https://wdm-a.wbx2.com/wdm/api/v1                   |
-| `WHISTLER_API_SERVICE_URL`           | The url to the whistler test service                                                                                     | https://whistler-prod.allnint.ciscospark.com/api/v1 |
+| `WDM_SERVICE_URL`                    | The WDM service URL before the catalog is downloaded                                                                     | https://wdm-a.wbx2.com/wdm/api/v1                   |
+| `WHISTLER_API_SERVICE_URL`           | The URL to the whistler test service                                                                                     | https://whistler-prod.allnint.ciscospark.com/api/v1 |
 | `WHISTLER`                           | Run (meetings) tests using Whistler users                                                                                | FALSE                                               |
 | `JENKINS`                            | Run specific tests that should be run on (internal) Jenkins                                                              | FALSE                                               |
 
 ### Running Tests
 
-`yarn run test` is the entrypoint to our test runner, but its not practical to use without parameters; the full suite would take over two hours to run and cross talk would probably cause tests to break each other.
-
-> Get the full test-runner docs via `yarn run test --help`.
+Since we are now using yarn workspaces, use the `yarn workspace "plugin name" run test:unit/browser/integration` to run tests for a specific package. This is practical and saves time.
```

Comment:
Can we have an example here for three different commands like below,
```suggestion
Since we are now using yarn workspaces, use the `yarn workspace "plugin name" run test:unit/browser/integration` to run tests for a specific package. This is practical and saves time.

Example:
- yarn workspace @webex/plugin-meetings test:unit
- yarn workspace @webex/plugin-meetings test:browser
- yarn workspace @webex/plugin-meetings test:integration
```

Reply:
Will do